client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -58,8 +58,9 @@ const RegisteredDomain = React.createClass( {
 			transferPath = paths.domainManagementTransfer( slug, name );
 
 		if ( hasPrivacyProtection ) {
+			let label;
 			if ( privateDomain ) {
-				return this.getLabel( {
+				label = this.getLabel( {
```

Comment:
I think it's best to just `return` here the result of `this.getLabel`, it will be more consistent with [the end of the function](https://github.com/Automattic/wp-calypso/pull/4194/files#diff-0af9958ef44ea186ebcdc3c86fb24c26R84).